README.md

```diff
@@ -86,10 +87,12 @@ use Rack::Attack
 
 __IMPORTANT__: By default, rack-attack won't perform any blocking or throttling, until you specifically tell it what to protect against by configuring some rules.
 
-## Usage
+## Usage examples
 
-*Tip:* The example in the wiki is a great way to get started:
-[Example Configuration](https://github.com/kickstarter/rack-attack/wiki/Example-Configuration)
+If you just want to get going asap, then you can take our [example configuration](docs/example_configuration.md)
+and tailor it to your needs using the [DSL reference](#reference) or check out the [advanced configuration](docs/advanced_configuration.md) examples.
+
+## Reference
```

Comment:
Not sure about breaking up Usage in two separate sections.
Please let's keep it as one section, and the links to the examples still as a "Tip".

I don't want to drive a lot of focus to the examples, until we update/refactor them.